### DISCUSSION TITLE
Stupid noob question

### DISCUSSION THREAD

**Discussion by Florida-Duke** [tteck/Proxmox#1645] [Q&A]
First thanks for some great scripts.
I'm sure that you know that they are referenced all over the web.
I'm very new to proxmox, Docker and Portainer.
I have installed all three using your script with ubuntu and removed it.
Then I tried your script with alpine.
My stupid problem is I can't seem to login to them via the web browser or the console.
Any information about the default user and password and/or the ports to use via the web browser.
Would be a big help.
I have googled till my eyes are crossed.
It seems that everyone has their own opinion about which way things should be done.
Thanks Again,

**Comment by tteck** [comment]
Portainer Interface: IP:9000

**Comment by X-Anonymous-Y** [comment]
What do you mean by:

"I have installed all three using your script with ubuntu and removed it."??? <-- Proxmox is a bare metal hypervisor and shouldn't run under Ubuntu anyway because Debian packages are required here.

Did you install Proxmox as substructure and then the "Docker" container with the "Portainer" container?

"My stupid problem is I can't seem to login to them via the web browser or the console." <-- Try the advanced installation and assign a password there
